deploy/helm/metacontroller/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 version: v2.2.4
 appVersion: v2.2.4
```

Comment:
query: Do we intend to keep both helm & app version same?
query: Are these auto magically updated every time a new release is cut (which again is automatic)?

Reply:
1. dunno, will check that, however the artifact name is determined from `version`
2. yes, they are updated by semantic-release on every release automatically